src/components/button/button.component.ts

```diff
@@ -46,7 +46,7 @@ export class OsButton implements IOsButton {
   }
 
   setDisabled() {
-    if (this.disabled) {
+    if (this.disabled || this.loading) {
```

Comment:
Does this change the interface contract? It does make sense to have it, but we should point it out in release notes.